docs/source/capabilities/algorand-client.md

```diff
@@ -48,38 +43,195 @@ kmd_client = algorand.client.kmd
+- `rekey_to: Optional[str]` - Change the signing key of the sender to the given address. **Warning:** Please be careful and read the [official rekey guidance](https://developer.algorand.org/docs/get-details/accounts/rekey/)
+- `note: Optional[bytes | str]` - Note to attach to transaction (UTF-8 encoded if string). Max 1000 bytes
+- `lease: Optional[bytes | str]` - Prevent duplicate transactions with same lease (max 32 bytes). [Lease documentation](https://developer.algorand.org/articles/leased-transactions-securing-advanced-smart-contract-design/)
+- Fee management:
+  - `static_fee: Optional[AlgoAmount]` - Fixed transaction fee (use `extra_fee` instead unless setting to 0)
+  - `extra_fee: Optional[AlgoAmount]` - Additional fee to cover inner transactions
+  - `max_fee: Optional[AlgoAmount]` - Maximum allowed fee (prevents overspending during congestion)
+- Validity management:
+
+  - `validity_window: Optional[int]` - Number of rounds transaction is valid (default: 10)
+  - `first_valid_round: Optional[int]` - Explicit first valid round (use with caution)
+  - `last_valid_round: Optional[int]` - Explicit last valid round (prefer `validity_window`)
+
+- [`SendParams`](todo_paste_url)
+  - `max_rounds_to_wait_for_confirmation: Optional[int]` - Maximum rounds to wait for confirmation
+  - `suppress_log: bool` - Suppress log messages (default: False)
+  - `populate_app_call_resources: bool` - Auto-populate app call resources using simulation (default: from config)
+  - `cover_app_call_inner_transaction_fees: bool` - Automatically cover inner transaction fees via simulation
+
+Some more transaction-specific parameters extend these base types:
+
+#### Payment Transactions (`PaymentParams`)
+
+- `receiver: str` - Recipient address
+- `amount: AlgoAmount` - Amount to send
+- `close_remainder_to: Optional[str]` - Address to send remaining funds to (for account closure)
+
+#### Asset Transactions
+
+- `AssetTransferParams`: Asset transfers including opt-in
+- `AssetCreateParams`: Asset creation
+- `AssetConfigParams`: Asset configuration
+- `AssetFreezeParams`: Asset freezing
+- `AssetDestroyParams`: Asset destruction
+
+#### Application Transactions
+
+- `AppCallParams`: Generic application calls
+- `AppCreateParams`: Application creation
+- `AppUpdateParams`: Application update
+- `AppDeleteParams`: Application deletion
+
+#### Key Registration
+
+- `OnlineKeyRegistrationParams`: Register online participation keys
+- `OfflineKeyRegistrationParams`: Take account offline
+
+Usage example with `AlgorandClient`:
+
+```python
+# Create transaction
+payment = client.create_transaction.payment(
+    PaymentParams(sender=account.address, receiver=receiver, amount=AlgoAmount(1))
+)
+
+# Send transaction
+result = client.send.send_transaction(payment, SendParams())
```

Comment:
I don't think that is valid.
I'm guessing we want something like this? `algorand.new_group().add_transaction(payment_txn).send()`